packages/eslint-config-next/package.json

```diff
@@ -18,20 +18,14 @@
     "publish-manual": "npm publish"
   },
   "peerDependencies": {
-    "@mintlify/eslint-config": "1.0.3",
-    "@mintlify/eslint-config-typescript": "1.0.8",
-    "@typescript-eslint/eslint-plugin": "5.x",
-    "@typescript-eslint/parser": "5.x",
+    "@mintlify/eslint-config": "1.0.4",
+    "@mintlify/eslint-config-typescript": "1.0.9",
+    "@typescript-eslint/eslint-plugin": "6.x",
+    "@typescript-eslint/parser": "6.x",
     "eslint": "8.x",
     "eslint-config-next": "13.x",
     "eslint-config-prettier": "8.x",
-    "eslint-config-react-app": "7.x",
-    "eslint-plugin-flowtype": "4.x",
-    "eslint-plugin-import": "2.x",
-    "eslint-plugin-jsx-a11y": "6.x",
-    "eslint-plugin-react": "7.x",
-    "eslint-plugin-react-hooks": "4.x",
```

Comment:
to my knowledge, none of these were being used. `eslint-config-next` uses some of them, but it also declares them as dependencies, so we don't need them as peer deps